command/daemon/daemon.go

```diff
@@ -100,7 +100,7 @@ func (c *daemonCommand) run(*kingpin.ParseContext) error {
 		),
 	)
 	store := database.ProvideInstanceStore(db)
-	poolManager := drivers.New(ctx, store, env.Settings.LiteEnginePath, env.Runner.Name)
+	poolManager := drivers.New(ctx, store, env.Settings.LiteEnginePath, env.Runner.Name, false)
```

Comment:
This should be an environment variable and injected in through the config struct. I have no context of what false is from looking at this. It would also mean you are touching less places in code as the config struct in injected up the layers anyways.

Reply:
That was my initial approach where I added IsHarness to EnvConfig and set config.IsHarness = true for CIE and not set it for others. That also required adding IsHarness to execCommand in exec.go and not set it. Please let me know if that approach works. I'll upload a new patch.